RxSwift/Disposables/Disposables.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Disposables.swift
+//  Rx
+//
+//  Created by Mohsen Ramezanpoor on 01/08/2016.
+//  Copyright © 2016 Mohsen Ramezanpoor. All rights reserved.
+//
+
+import Foundation
+
+public struct Disposables {
+    
+}
+
+public extension Disposables {
+    
+    static func empty() -> Disposable {
```

Comment:
Ok, so let's change this to `create` like we've discussed on slack. We also need to add inline documentation to all of these methods, and probably make them public? 

I think we are also missing a variadic version (3 disposables + ...).